apps/application-system/api/src/app/modules/application/tools/application.serializer.ts

```diff
@@ -112,6 +112,8 @@ export class ApplicationSerializer
           application,
           userRole,
           intl.formatMessage,
+          undefined,
+          nationalId,
```

Comment:
Change the order of parameters so that we can skip passing undefined to the function

```ts
helper.getCurrentStatePendingAction(
  application,
  userRole,
  intl.formatMessage,
  nationalId,
)
```